### ISSUE TITLE
Title: WCAG 4.1.2: Ensures every form element has a label (.PrivateSwitchBase-input-317[value=""][data-indeterminate="false"]) 

### ISSUE THREAD

**Issue by mihirprakash** [mbrn/material-table#3247]
Issue: Ensures every form element has a label (label - https://accessibilityinsights.io/info-examples/web/label)

Target application: Luminate Portal - http://localhost:7777/local-dev/plans/663b1af80f7ed815d83846bf/shipments

Element path: #mfe;.PrivateSwitchBase-input-317[value=""][data-indeterminate="false"]

Snippet:

How to fix:
Fix any of the following:
Form element does not have an implicit (wrapped)
Form element does not have an explicit
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element has no placeholder attribute
Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Chrome version 124.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Version:    "material-table": "^2.0.5",
